ModManager_Classes/GithubIntegration/GithubDownloader.cs

```diff
@@ -8,12 +8,22 @@
 
 namespace Imya.GithubIntegration
 {
+    public struct DownloadResult
+    {
+        public DownloadResult() { }
+
+        public bool DownloadSuccessful = false;
+        public String DownloadDestination = String.Empty;
+    }
+
     public class GithubDownloader
     {
         private readonly String DOWNLOAD_DIRECTORY;
 
         private static GitHubClient GithubClient = new GitHubClient(new ProductHeaderValue("iModYourAnno"));
 
+        public static int DownloadBufferSize { get; } = 81920;
```

Comment:
Does this restrict the dl to 80 Mb?

Reply:
No, it's the buffer size for the download streamcopy.

The copy method works like this: 

while DownloadStream has bytes left
read buffersize amount of bytes from the source stream (http response)
copy buffer to target stream (filestream)

Reply:
all good then